systers_portal/templates/account/logout.html

```diff
@@ -0,0 +1,21 @@
+{% extends "base.html" %}
+{% load url from future %}
+{% load i18n %}
+{% block head_title %}{% trans "Sign Out" %}{% endblock %}
```

Comment:
`head_title` is a block from django-allauth, in contrast we are using `title` block for page titles.
Here is an example from signup template:
`{% block title %} - Sign Up{% endblock %}`